docs/src/main/asciidoc/security-oidc-bearer-authentication-concept.adoc

```diff
@@ -915,6 +915,6 @@ Note Quarkus `web-app` applications always require `quarkus.oidc.client-id` prop
 * xref:security-openid-connect-client-reference.adoc[OpenID Connect and OAuth2 Client and Filters Reference Guide]
 * xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak]
 * xref:security-jwt-build.adoc[Sign and encrypt JWT tokens with SmallRye JWT Build]
-* xref:security-overview-concept.adoc#oidc-jwt-oauth2-comparison[Summary of Quarkus OIDC, JWT and OAuth2 features]
+* xref:security-authentication-mechanisms.adoc#combining-authentication-mechanisms[Combining authentication mechanisms]
```

Comment:
This file has links to the OIDC web authentication at the very top in the introduction

Reply:
If you'd like please keep the new `Combining authhentication mechanisms` as well, but IMHO, a link to the comparison of the token verification options should remain on its own